br/pkg/task/restore.go

```diff
@@ -168,6 +170,7 @@ func DefineRestoreCommonFlags(flags *pflag.FlagSet) {
 		"(deprecated) concurrency pd-relative operations like split & scatter.")
 	flags.Uint(FlagStatsConcurrency, defaultStatsConcurrency,
 		"concurrency to restore statistic")
+	flags.Bool(FlagAutoAnalyze, false, "trigger tidb analyze priority queue to analyze table")
```

Comment:
lightning will always update stats_meta, do you think we can set the default value to true?

Reply:
Sounds good. Because the default behavior in previous versions of BR is that set `modify_count` to 0, need a document and release note to notice customers who manually analyze tables after restore.